Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM quay.io/prometheus/busybox:latest
+# pin busybox version
```

Comment:
No need for this. Maybe it would be rather interesting to comment on data of which this image is from (:

Reply:
comment https://quay.io/repository/prometheus/busybox?tab=tags?

Reply:
Still, I can't really see that certain sha256 there :thinking:

Reply:
As commit said, you can see sha256 in https://quay.io/repository/prometheus/busybox?tab=history, i use the latest tag at Nov 13, 2019.

Reply:
https://quay.io/repository/prometheus/busybox/manifest/sha256:598bed4d1a87712e9d5e5863c0458c20161362d4f41233ee262e44ae69f21b33

Manifest not found ):